packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -2223,13 +2247,7 @@ abstract class MultiSelectableSelectionContainerDelegate extends SelectionContai
     if (selections.isEmpty) {
       return null;
     }
-    final StringBuffer buffer = StringBuffer();
-    for (final SelectedContent selection in selections) {
-      buffer.write(selection.plainText);
-    }
-    return SelectedContent(
-      plainText: buffer.toString(),
-    );
+    return SelectedContent(plainText: copyInterceptor.intercept(selections));
```

Comment:
Is this all there is to how we currently do copy? If so, I kind of like this approach at a high level. Just give users the ability to use their own function at this point, so they can pretty much do whatever they want here.